addons/udes_stock/views/edi_menu_views.xml

```diff
@@ -6,5 +6,9 @@
     <menuitem id="edi_record_menu" name="UDES"
 	      parent="edi.record_menu" sequence="500"/>
 
+    <!-- EDI move tracker menu item -->
```

Comment:
does udes_stock depennd on edi? if not this needs to live somewhere else.

Reply:
it does!